cluster/ubuntu/util.sh

```diff
@@ -44,27 +47,28 @@ function setClusterInfo() {
   for i in $nodes; do
     nodeIP=${i#*@}
 
-    if [[ "${roles[${ii}]}" == "ai" ]]; then
+    local n_role="${roles[${ii}]}"
+    if [[ $n_role == "ai" ]]; then
```

Comment:
nit: `"${n_role}"` (and below). I know it's not necessary, but https://google.github.io/styleguide/shell.xml#Quoting